### ISSUE TITLE
Temporary Welcome page

### ISSUE THREAD

**Issue by michalspano** [MSQ-Consulting/msqconsulting.github.io#1]
Consider adding a __temporary__ welcome message at the root of the site. Hence, contact methods are viable (even if the _design_ of the website is currently under development).

Example:

```txt
Page Header
---
Business contact: foo
Private contact: bar
Support: baz
```

**Comment by michalspano** [comment]
\+ inform that the site is __under maintenance/development break__.

**Comment by michalspano** [comment]
Fix: v0.0.1

**Comment by michalspano** [comment]
Fix v0.0.0

**Comment by michalspano** [comment]
Fix: 457600154b58b1cdc71f05f65a29d10e8a3890e7
